saleor/graphql/account/mutations.py

```diff
@@ -1,9 +1,29 @@
 import graphene
+from django.conf import settings
 from django.contrib.auth.tokens import default_token_generator
+from django.contrib.sites.models import Site
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+from graphql_jwt.decorators import permission_required
 
-from ...account import models
+from ...account import emails, models
 from ...core.permissions import MODELS_PERMISSIONS, get_permissions
-from ..core.mutations import ModelMutation
+from ..core.mutations import BaseMutation, ModelMutation
+from ..core.types import Error
+
+
+def send_user_password_reset_email(user):
+    site = Site.objects.get_current()
```

Comment:
I just realized that we can save this database query. We have the middleware `saleor.core.middleware.site` that fetches the current `site` and assigns it to the `request`. In Graphene we can access it in resolvers: `info.context.site` so we would have to pass it to this function as a second argument.